IM_calculation/Advanced_IM/Models/Burks_Baker_2013/run.py

```diff
@@ -60,18 +62,23 @@ def main(comp_000: Path, comp_090: Path, rotd: bool, output_dir: Path):
 
     accelerations = np.array((waveforms["000"], waveforms["090"])).T
     ordered_columns_dict = {}
+    sd_rotd_list = []
+
 
     for z in z_list:
         for dt in dt_list:
             for alpha in alpha_list:
                 for dy in dy_list:
```

Comment:
TODO: replace the quad for loop with itertools.product (Would reduce the indentation by 3 levels)
Not needed for this PR, could add in the TODO comment though